src/core/math/vec3/unit.test.js

```diff
@@ -1,56 +1,48 @@
 const test = require('ava')
 const {unit, fromValues, toString} = require('./index')
 
+const {compareVectors} = require('../../../../test/helpers/index')
```

Comment:
very cool!
btw you can leave out the index btw, node looks for 'index.js' files in folders directly, so you can just use const {compareVectors} = require('../../../../test/helpers')